python/python/tests/test_async_client.py

```diff
@@ -1457,6 +1457,28 @@ async def test_rename(self, redis_client: TRedisClient):
                 "{same_slot}" + "non_existing_key", "{same_slot}" + "_rename"
             )
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_renamenx(self, redis_client: TRedisClient):
+        key1 = f"{{testKey}}:1-{get_random_string(10)}"
+        key2 = f"{{testKey}}:2-{get_random_string(10)}"
+        key3 = f"{{testKey}}:3-{get_random_string(10)}"
+        string_key = f"{{testKey}}:4-{get_random_string(10)}"
+        non_existing_key = f"{{testKey}}:5-{get_random_string(10)}"
+
+        with pytest.raises(RequestError):
+            assert await redis_client.renamenx(non_existing_key, key1)
+
+        assert await redis_client.set(key1, "foo") == OK
```

Comment:
these tests were updated recently.  Would you mind taking a look?  
https://github.com/aws/glide-for-redis/pull/1483/files#diff-7fdec9c1136630b21338f81bbd06c578baca837e3079e1ba5d848f112fab2e2eR2385